lib/tokenizers.js

```diff
@@ -259,9 +279,19 @@ export async function _createTokenizer() {
   }
 
   // 4. Create a keystore and HMAC key for the tokenizer.
-  const keystore = await kms.createKeystore({
-    controller: capabilityAgent.id, referenceId: 'primary'
-  });
+  let keystore;
+  try {
+    keystore = await kms.createKeystore({
+      controller: capabilityAgent.id, referenceId: 'primary'
+    });
+  } catch(cause) {
+    const error = new BedrockError(
```

Comment:
Once this PR lands: https://github.com/digitalbazaar/webkms-client/pull/32 ... I don't think this is going to be that helpful of an error message. Instead, let's wrap this whole function in a try/catch and throw an error that says we failed to create a tokenizer and set the cause accordingly.

Reply:
Yeah, good point.

Reply:
Moved to wrap the whole function.